CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Push to GCP app collection
+
```

Comment:
I'd remove that from the changelog. From the "user" perspective it doesn't change a thing.

Reply:
Not really, you can put the manifest in the collection manually. In my view users are not concerned with CD but rather with the functionality.